verification/src/transaction_verifier.rs

```diff
@@ -24,13 +25,13 @@ where
     pub fn new(
         rtx: &'a ResolvedTransaction,
         median_time_context: &'a M,
-        tip_number: BlockNumber,
+        block_number: BlockNumber,
         tip_epoch_number: EpochNumber,
```

Comment:
Suggest correcting the variable name `tip_epoch_number` together, and the type should be `EpochNumber`.